Markup/Editor/css/custom.css

```diff
@@ -0,0 +1,128 @@
+@charset "UTF-8";
+/*
+~   Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
+~
+~   Licensed under the Apache License, Version 2.0 (the "License");
+~   you may not use this file except in compliance with the License.
+~   You may obtain a copy of the License at
+~
+~        http://www.apache.org/licenses/LICENSE-2.0
+~
+~   Unless required by applicable law or agreed to in writing, software
+~   distributed under the License is distributed on an "AS IS" BASIS,
+~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~   See the License for the specific language governing permissions and
+~   limitations under the License.
+*/
+
+ul li, ul {list-style: none;margin: 0;padding: 0}
+a:hover, a:focus {
+    text-decoration: none;
+}
+
+.nav-tabs > li > a {
+ color: rgba(255, 255, 255, 0.5);
+ -webkit-transition: color 250ms ease;
+ -moz-transition: color 250ms ease;
+ transition: color 250ms ease;
+}
+.nav-tabs > li {margin-bottom:0;}
+.nav > li > a:hover, .nav > li > a:focus { 
+    color: #ffffff;
+    background-color: rgba(255,255,255,0.25);
+    border-color: transparent;
+}
+.nav-tabs {
+    background-color: #353535;
+    color: #fff;
+    border: 0
+}
+.nav-tabs > li.active > a, .nav-tabs > li.active > a:hover, .nav-tabs > li.active > a:focus {
+    background-color: #222;
+    border-color: #222;
+    border-top-color: #ff800f;
+    color: #fff
+}
+.tab-content .content {
+    padding: 15px
+}
+.sidebar-left {
+    color: #fefefe;
+}
+.tab-content .content h3 { 
+    margin-bottom: 20px;
+    margin-top: 5px;
+}
+.tab-content button.close {
+    padding-left: 10px;
+    display:none;
+}
+.tab-content li.active button.close {
+    display: block;
+}
+.tab-content .nav > li > a {
+    display: block;
+    position: relative;
+}
+.add-sub-tab a {padding: 10px}
+
+
+/* debug */
+
+.debugger-toolbar {
+	line-height: 20px;
+	padding: 5px 10px;
+	background: #282B2F
+}
+
+.debugger-toolbar span {
+	display: inline-block;
+	font-size: 16px;
+	font-weight: 500;
+	vertical-align:middle; 
+}
+
+.debugger-toolbar a {
+	padding: 2px;
+	margin: 0 2px;
+	font-size: 20px
+}
+
+.debugger-toolbar span.appName {
+	padding-right: 25px;
+	color: #d6d6d6
+}
+
+
+.close-handle {
+    background: #3d464f none repeat scroll 0 0;
+    border: medium none;
```

Comment:
border: border-width border-style border-color|initial|inherit; 

If border none why we are setting medium?